src/modules/languages/rust.nix

```diff
@@ -60,6 +60,10 @@ in
       pre-commit.tools.rustfmt = lib.mkForce cfg.packages.rustfmt;
       pre-commit.tools.clippy = lib.mkForce cfg.packages.clippy;
     })
+    (lib.mkIf pkgs.stdenv.isDarwin {
+      env.RUSTFLAGS = [ "-L framework=${config.env.DEVENV_PROFILE}/Library/Frameworks" ];
+      env.RUSTDOCFLAGS = [ "-L framework=${config.env.DEVENV_PROFILE}/Library/Frameworks" ];
```

Comment:
My haskell project broke after an upgrade of devenv (using flake). I tracked the failure down to an infinite recursion introduced by this line on Darwin.
@onsails can you have another look at this fix?

Reply:
Could you provide a minimal example to reproduce?

Reply:
Fixed in master, thanks!